app/components/works/contact_email_row_component.html.erb

```diff
@@ -8,7 +8,7 @@
     <div class="invalid-feedback">You must provide a valid email address</div>
   </div>
   <div class="col-sm-1">
-    <%= button_tag type: "button", class: "btn btn-sm float-end", aria: {label: "Remove"},
+    <%= button_tag type: "button", class: "btn btn-sm float-end", aria: {label: "Remove Association"},
```

Comment:
"Remove Association" might strike the user as vague.  What's being removed here?

Reply:
Funny that "Remove" was dinged as not meaningful...

Reply:
Would something like `Remove contact email` be appropriate?